src/server/game/AI/CoreAI/CombatAI.cpp

```diff
@@ -54,9 +54,9 @@ void AggressorAI::UpdateAI(uint32 /*diff*/)
 
 void CombatAI::InitializeAI()
 {
-    for (uint32 i = 0; i < MAX_CREATURE_SPELLS; ++i)
-        if (me->m_spells[i] && sSpellMgr->GetSpellInfo(me->m_spells[i]))
-            Spells.push_back(me->m_spells[i]);
+    for (uint32 m_spell : me->m_spells)
```

Comment:
I don't think the iterator should have m_ in the name, since that stands for member.

Reply:
It indeed should be `l_Spell` 😄 or just `spell` since Shauren hates this code style.

Reply:
True, I'll update.